NSS-Day-Cohort-42/NSS-Day-Cohort-42.github.io
# Replaced old classmate photos with C42 headshots.

Related to issue #6 

## Description of Proposed Changes

- Removed previous cohort's photos from the classmate photos folder and the group photos folder.
- Uploaded new group photo for C42 and all available headshots from C42 Google Drive.

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout mp-photos
```
1. Review photo files and make sure no duplicates are present.
